components/file_upload/file_upload.jsx

```diff
@@ -242,6 +242,8 @@ class FileUpload extends React.PureComponent {
         if (typeof files !== 'string' && files.length) {
             this.uploadFiles(files);
         }
+
+        this.props.onFileUploadChange();
```

Comment:
https://github.com/mattermost/mattermost-webapp/pull/782/files - I reverted change on that PR. Found out that the real cause of calling file upload change was during register drag events. I moved `props.onFileUploadChange()` to `handleDrop` instead.

Reply:
@lindalumitchell Once merged, may I request your test/verification to tickets covered by  https://github.com/mattermost/mattermost-webapp/pull/782?
- MM-9007
- MM-9008
- MM-9013